tests/integration/helpers.py

```diff
@@ -251,6 +269,41 @@ async def wait_for(
     raise TimeoutError()
 
 
+async def generate_client_from_application(
+    ops_test: OpsTest, jenkins_app: Application
+) -> UnitWebClient:
+    """Generate a Jenkins client directly from the Juju application, without fixtures.
+
+    Args:
+        ops_test: OpsTest framework
+        jenkins_app: Juju Jenkins-k8s application.
+
+    Returns:
+        A Jenkins web client.
+    """
+    assert ops_test.model
+    address = await get_model_jenkins_unit_address(ops_test.model, jenkins_app.name)
+    jenkins_unit = jenkins_app.units[0]
+    # pylint: disable=duplicate-code
```

Comment:
I don't think you need to disable this anymore

Reply:
I've tested and it is necessary still :(